src/Form.tsx

```diff
@@ -64,6 +64,23 @@ const Form = (
     setFormData(newFormData);
   }
 
+  function mod(n: number, m: number) {
```

Comment:
Just so you know I'll probably move this to a helper file at some point. right now we just have a color.ts file and a searchHelpers.ts and I need to structure/name the helpers better so I'm ok that this is here now.